schema-sandbox/schema.json

```diff
@@ -47,6 +77,38 @@
         "authors"
     ],
     "definitions": {
+        "date": {
+            "format": "date",
+            "pattern": "[0-9]{4}-[0-1]{1}[0-9]{1}-[0-3]{1}[0-9]{1}",
+            "type": "string"
+        },
+        "doi": {
+            "pattern": "^10\\.\\d{4,9}(\\.\\d+)?/[A-Za-z0-9-\\._;\\(\\)\\[\\]\\\\\\\\:/]+$",
+            "type": "string"
+        },
+        "identifier": {
+            "additionalProperties": false,
+            "properties": {
+                "type": {
+                    "enum": [
+                        "doi",
+                        "other",
+                        "swh",
+                        "url"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "minLength": 1,
+                    "type": "string"
+                }
+            },
```

Comment:
We should add doi validation using `#/definitions/doi` for the case when an identifier happens to be a doi, I'll make an issue